extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -245,7 +259,8 @@ export class SubscriptionService {
     return this.viewer_.whenFirstVisible().then(() => {
       return this.timer_.timeoutPromise(
           timeout,
-          subscriptionPlatform.getEntitlements()
+          subscriptionPlatform.getEntitlements(
+              this.serviceAdapter_.getEncryptedDocumentKey())
```

Comment:
We no longer have `encryptedKey` in the `SubscriptionPlatform.getEntitlements` call, right? It looks like `srviceAdapater.getEncryptedKey()` is intended to be used inside the implementation.

Otherwise, `getEncryptedKey()` requires `serviceId` param.